jerry-core/jerry.c

```diff
@@ -1766,7 +1766,7 @@ jerry_run (jerry_api_value_t *error_value_p) /**< [out] error value */
   ecma_value_t error_value;
   jerry_completion_code_t ret_code = vm_run_global (&error_value);
 
-  error_value &= ~(1u << ECMA_VALUE_ERROR_POS);
+  error_value &= ~ECMA_VALUE_ERROR_FLAG;
```

Comment:
Well, it is in the original code. I think we can remove this in a follow-up patch if unnecessary. It would be better for tracking issues.